pyproject.toml

```diff
@@ -1,3 +1,48 @@
+[project]
+requires-python = "== 3.12"
+
+# Bumpversion configuration
+
+[tool.bumpversion]
+commit = false
+tag = false
+current_version = "3.1.0" 
+
+# Defines the files to be updated.
+[[tool.bumpversion.files]]
+filename = "src/objects/__init__.py"
+
+[[tool.bumpversion.files]]
+filename = "README.rst"
+search = ":Version: {current_version}"
+replace = ":Version: {new_version}"
+
+[[tool.bumpversion.files]]
+filename = "README.NL.rst"
+search = ":Version: {current_version}"
+replace = ":Version: {new_version}"
+
+[[tool.bumpversion.files]]
+filename = "package.json"
+search = '"version": "{current_version}"'
+replace = '"version": "{new_version}"'
```

Comment:
see https://github.com/GPP-Woo/GPP-publicatiebank/blob/3562db1150f1402aad6db4122a4c45e2af5505d4/pyproject.toml#L37 for an example of how to make sure that `package-lock.json` is also updated, and in general there are a bunch of useful configuration options for bump-my-version :)